### ISSUE TITLE
freq='7D' in DeepAR

### ISSUE THREAD

**Issue by dai-ichiro** [awslabs/gluon-ts#464]
I'm using DeepAR. The frequency of my data is '7D'.
This is my code.
```
from gluonts.dataset.common import ListDataset
from gluonts.model.deepar import DeepAREstimator
from gluonts.trainer import Trainer
from gluonts.evaluation.backtest import make_evaluation_predictions


dataset = ListDataset(
    [
        {"target": [3, 4, 7, 6, 5], "start": "2019-7-2"}
    ],
    freq="7D"
)

estimator = DeepAREstimator(freq="7D",
                            prediction_length=3, 
                            trainer=Trainer(
                                epochs=1,
                                 ),
                            )

predictor = estimator.train(training_data=dataset)

forecast_it, ts_it = make_evaluation_predictions(
    dataset=dataset,  
    predictor=predictor,  
    num_samples=100,
)

forecasts = list(forecast_it)

print(forecasts[0].start_date)
```
This code returns '2019-07-11 00:00:00'
Why? I'm confused.
When I use 'feat_dynamic_real',  start_date of forecast result is very important, I think.

How should I change my code?
I would be grateful if anyone could tell me.

**Comment by jaheba** [comment]
There are a few things at play here:

* We align timestamps. The idea is that all timestamps within the same range, are represented by the same value. In this case `2019-7-2` is aligned to `2019-06-27`.

* `make_evaluation_predictions` is not the right method to just make predictions. The idea is to have a separate dataset, we want to test the accuracy of the model against.

* `make_evaluation_predictions` assumes that the provided time-series are of shape `observed-data + prediction_length`. Since `prediction_length` is `3` and the total length of your `target` is `5`, we get the first prediction after two observations. `2019-06-27` + `14 days` is `2019-07-11`

I hope that clarifies the confusion.

---

Notes to @awslabs/gluon-ts :

* We use `floor` instead of `ceil`, but `rollforward` instead of `rollback`. Should we align this logic?

* I think we should remove `make_evaluation_predictions` entirely for a better more understandable solution.

**Comment by dai-ichiro** [comment]
Thank you very much for your quick reply. Your explanation clarified my confusion.
